### PR TITLE
JAMES-3262 mailbox inconsistencies header

### PR DESCRIPTION
**Pull request by Arsnael**

### CODE REVIEW

**Comment by Arsnael** [comment]
```
[2ddfaf42f9006c8d2d68e20b5be38be7b0d75b3d] [ERROR] Failures: 
[2ddfaf42f9006c8d2d68e20b5be38be7b0d75b3d] [ERROR]   MemoryTerminationSubscriberTest.dynamicListeningEventsShouldGetOnlyNewEvents 
[2ddfaf42f9006c8d2d68e20b5be38be7b0d75b3d] Expecting:
[2ddfaf42f9006c8d2d68e20b5be38be7b0d75b3d]   <[Cancelled(TaskAggregateId(TaskId{value=90ee1650-32da-4406-aab6-f8177287035d}),EventId(42),None)]>
[2ddfaf42f9006c8d2d68e20b5be38be7b0d75b3d] to contain exactly (and in same order):
[2ddfaf42f9006c8d2d68e20b5be38be7b0d75b3d]   <[Failed(TaskAggregateId(TaskId{value=cf7d1ace-d40c-48e7-ab04-64fee8e487e5}),EventId(42),None,None,None),
[2ddfaf42f9006c8d2d68e20b5be38be7b0d75b3d]     Cancelled(TaskAggregateId(TaskId{value=90ee1650-32da-4406-aab6-f8177287035d}),EventId(42),None)]>
[2ddfaf42f9006c8d2d68e20b5be38be7b0d75b3d] but could not find the following elements:
[2ddfaf42f9006c8d2d68e20b5be38be7b0d75b3d]   <[Failed(TaskAggregateId(TaskId{value=cf7d1ace-d40c-48e7-ab04-64fee8e487e5}),EventId(42),None,None,None)]>
```
test this please

File: src/site/markdown/server/manage-webadmin.md [hunk 1]
```diff
@@ -412,13 +412,15 @@ The kind of task scheduled depends on the action parameter. See below for detail
 This task is only available on top of Guice Cassandra products.
 
 ```
-curl -XPOST /mailboxes?task=SolveInconsistencies
```
Comment (chibenwa) [inline src/site/markdown/server/manage-webadmin.md:415]: If I recall well, laconic documentation here was intentionnal 

cc @mbaechler

File: src/site/markdown/server/manage-webadmin.md [hunk 2]
```diff
@@ -412,13 +412,15 @@ The kind of task scheduled depends on the action parameter. See below for detail
...
+curl -XPOST -H "I-KNOW-WHAT-I-M-DOING: ALL-SERVICES-ARE-OFFLINE" /mailboxes?task=SolveInconsistencies
 ```
 
 Will schedule a task for fixing inconsistencies for the mailbox deduplicated object stored in Cassandra.
 
 [More details about endpoints returning a task](#Endpoints_returning_a_task).
 
+The `I-KNOW-WHAT-I-M-DOING` header is mandatory (you can read more information about it in the warning section below).
```
Comment (chibenwa) [inline src/site/markdown/server/manage-webadmin.md:422]: IMO we can keep that one

**Comment by Arsnael** [comment]
Merged

**Pull request closed without merging**